### ISSUE TITLE
Use of 'starting' for processes needs disabiguation

### ISSUE THREAD

**Issue by mahermanns** [mpi-forum/mpi-issues#852]
# Problem

*Starting* is now a defined term for operations in context of procedure calls. The dynamic chapter uses the term in context of `mpiexec` and `mpirun`. We should clean up this ambiguous use of *start*.

# Proposal

Replace the language *start* by *launch* where approriate

# Changes to the Text

# Impact on Implementations

None.

# Impact on Users

Clearer use of language in the standard.

# References and Pull Requests

**Comment by mahermanns** [comment]
### Discussion on *launch* vs. *spawn*:

- Spawn adheres to the fact that 'spawned' processes behave slightly different (or have different characteristics)
- Both words describe how to add MPI processes to an job
- <del>Processes can be launched on different nodes, but spawned only on the local node</del> (Updated due to discussion)
- At some point the user can/should not distinguish between how the MPI processes came into being
- Launching and spawning have an impact on the inter- vs. intra-communicator (can be overcome by merging to an intra-communicator)

**Comment by mahermanns** [comment]
@wgropp Could be create some for of a index reference for **startup mechanism (see. *launch mechanism*)**

Because cleaning up the index would loose the existing index entries for *startup mechanism* or *startup*

**Comment by sonjahapp** [comment]
> * Processes can be launched on different nodes, but spawned only on the local node

I don't think this is true. One can actually specify the `host` as info key on `MPI_COMM_SPAWN(_MULTIPLE)` to specify a host to spawn to (See Section 11.8.4). An on managed resources (e.g. via Slurm) spawning usually works across the available resource allocation (i.e., nodes).

Furthermore, I could not find a place in the standard that enforces spawning to be limited the local node. If there is such a text somewhere in the standard I would appreciate if someone could point me to it.

**Comment by wgropp** [comment]
On the index, you can add a cross reference with the "|" character. Here's an example from the makeindex manual:
`\index{at!bat|see{bat, at}}` which gives `bat, see bat, at` (here for a second level index "bat" on "at". I'll have to check that the index processing works for this, but this shouldn't be a problem.
